pkg/chartverifier/report.go

```diff
@@ -50,16 +46,16 @@ type ReportMetadata struct {
 }
 
 type ToolMetadata struct {
-	Version                string `json:"verifier-version" yaml:"verifier-version"`
-	ChartUri               string `json:"chart-uri" yaml:"chart-uri"`
-	Digest                 string `json:"digest" yaml:"digest"`
-	LastCertifiedTimestamp string `json:"lastCertifiedTimestamp" yaml:"lastCertifiedTimestamp"`
-
-	CertifiedOpenShiftVersions string `json:"certifiedOpenShiftVersions" yaml:"certifiedOpenShiftVersions"`
+	Version                    string `json:"verifier-version" yaml:"verifier-version"`
+	Profile                    string `json:"profile-name" yaml:"profile-name"`
+	ChartUri                   string `json:"chart-uri" yaml:"chart-uri"`
+	Digest                     string `json:"digest,omitempty" yaml:"digest,omitempty"`
```

Comment:
Which cases `digest` is supposed to be empty?

Reply:
With the new profile the annotations to include are configurable, so this enables the report to omit the digest if it is not required by the profile.